src/status_im/contexts/communities/events.cljs

```diff
@@ -184,3 +187,28 @@
                      :params     []
                      :on-success #(rf/dispatch [:communities/fetched-collapsed-categories-success %])
                      :on-error   #(log/error "failed to fetch collapsed community categories" %)}]}))
+
+(rf/reg-event-fx :communities/share-community-channel-url-with-data
+ (fn [_ [chat-id]]
+   (let [{:keys [community-id channel-id]} (data-store.chats/decode-chat-id chat-id)]
+     {:json-rpc/call [{:method     "wakuext_shareCommunityChannelURLWithData"
+                       :params     [{:CommunityID community-id :ChannelID channel-id}]
+                       :on-success #(share/open
+                                     (if platform/ios?
+                                       {:activityItemSources [{:placeholderItem {:type "text"
+                                                                                 :content
+                                                                                 "Channel on Status"}
+                                                               :item            {:default {:type    "url"
+                                                                                           :content %}}
+                                                               :linkMetadata    {:title
+                                                                                 "Channel on Status"}}]}
+                                       {:title   "Channel on Status"
```

Comment:
I guess these strings should also be i18n labels too 👍

Reply:
👍🏻